contracts/src/v0.8/dev/vrf/VRFV2PlusWrapper.sol

```diff
@@ -384,11 +386,23 @@ contract VRFV2PlusWrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsume
     lastRequestId = requestId;
   }
 
+  function removeDomainSeparator(bytes memory bytesInput) public view returns (bytes memory newBytes) {
+    newBytes = new bytes(bytesInput.length);
+    for (uint8 i = 4; i < 36; ++i) {
+      newBytes[i - 4] = bytesInput[i];
+    }
+    return newBytes;
+  }
+
   function requestRandomWordsInNative(
     uint32 _callbackGasLimit,
     uint16 _requestConfirmations,
-    uint32 _numWords
+    uint32 _numWords,
+    bytes memory extraArgs
```

Comment:
Ditto on the `calldata` comment above, might apply even more here.

Reply:
Fixed this